pysteps/verification/salscores.py

```diff
@@ -155,11 +155,11 @@ def sal_structure(
     observation_objects = _sal_detect_objects(
         observation, thr_factor, thr_quantile, tstorm_kwargs
     )
-    prediction_volume = _sal_scaled_volume(prediction_objects).sum()
-    observation_volume = _sal_scaled_volume(observation_objects).sum()
+    prediction_volume = sal_scaled_volume(prediction_objects)
```

Comment:
why did you rename the calls to `_sal_scaled_volume`? this is causing the NameErrors in the tests, can you revert this change?